_unittest/example_models/T08/primitives_file.json

```diff
@@ -0,0 +1,79 @@
+{
+	"Primitives":
+	[
+		{
+			"Primitive Type": "Cylinder",
+			"Name": "Small",
+			"Plane": 1,
+			"Height": 10.0,
+			"Radius": "3.0mm",
+			"Internal Radius": 1.0,
+			"Number of Segments": 0
+		},
+		{
+			"Primitive Type": "Cylinder",
+			"Name": "Big",
+			"Plane": 0,
+			"Height": 50.0,
+			"Radius": 10.0,
+			"Internal Radius": 0
+		},
+		{
+			"Primitive Type": "Cylinder",
+			"Name": "Big_hole",
+			"Plane": 0,
+			"Height": 50.0,
+			"Radius": 10.0,
+			"Internal Radius": 5
+		}
+	],
+	"Instances":
+	[
+		{
+			"Name":"Small",
+			"Coordinate System": "CS1",
+			"Origin": ["0mm","1.052dm","2"]
+		},
+		{
+			"Name":"Small",
```

Comment:
@Samuelopez-ansys , Why do we have different key "Instances" ? can't we include the coordinate system and origin under the  "Primitives" itself ?

Reply:
@siva-krishnaswamy You first define the primitives you want to create, which has some properties, and then you can create as many instances of this primitive as you want, this approach is very powerful, because you can create for example 4 instances of the same primitive in your design.

Reply:
Thank you, @Samuelopez-ansys. Your explanation makes perfect sense, and I agree that this approach is highly versatile. I believe that we can further expand its scope to encompass CAD and 3D components. We could introduce definitions that reference the locations of CAD files and 3D component files. And instances could be used to place and orient CAD and 3Dcomponent.